viewer/packages/cad-model/src/picking/PickingHandler.ts

```diff
@@ -133,11 +189,9 @@ export class PickingHandler {
     if (treeIndex === undefined) {
       return undefined;
     }
-    const depth = await this.pickDepth(pickInput, async);
+    const perspectiveDepthNormalized = await this.pickDepth(pickInput, async);
```

Comment:
Below here is refactorings only to reduce code duplication for the Depth only raycast